garden_ai/gardens.py

```diff
@@ -1,7 +1,6 @@
 from __future__ import annotations
 
 import logging
-from typing_extensions import Self
```

Comment:
did we mean to take this out or did it just get dropped in a rebase or something? not too fussed either way just curious

Reply:
I was getting errors in the 3.10 build, it looks like `Self` was added in 3.11.